.eslintrc

```diff
@@ -18,6 +18,10 @@
 	"func-names": [2, "always"],
         "strict": [ "error", "global" ],
         "consistent-return": "off",
-        "no-loop-func": "off"
+        "no-loop-func": "off",
+        "padded-blocks": "off",
+        "no-multi-spaces": "off",
+        "no-useless-escape": "off",
+        "callback-return": "off"
```

Comment:
@richardkiene Do you intend to ever work through what I'm guessing are a number of eslint style nits in the current code from these rules?  I'm fine either way.

Reply:
@trentm Yeah, I've created [TRITON-1935](https://jira.joyent.us/browse/TRITON-1935) for bringing it all up to snuff with the latest plugin rules. Unfortunately the updated joyent eslint bits which were necessary to fix the security vulnerability also came with new eslint rules that had not been previously enforced.